private-templates-service/client/public/sample-catalogue.json

```diff
@@ -0,0 +1,25 @@
+[
```

Comment:
Can we move all of the designer files into their own folder so we know which is imported?